packages/toolpad-components/src/Text.tsx

```diff
@@ -34,7 +33,21 @@ function Text({ value, markdown, href, loading, mode, sx, ...rest }: TextProps)
       ) : (
         <MarkdownContainer>
           <React.Suspense>
-            <ReactMarkdown remarkPlugins={[remarkGfm]}>{value}</ReactMarkdown>
+            <ReactMarkdown
```

Comment:
We need the markdown component to have a min-height when it's empty so that developers can make changes.

<img width="814" alt="Screenshot 2022-12-20 at 11 43 16" src="https://user-images.githubusercontent.com/3165635/208659076-dbb4b94e-0682-4582-8602-9fc2519b59d0.png">

Reply:
We might want to handle `<code><pre>` and `<code>`. 

<img width="588" alt="Screenshot 2022-12-20 at 11 45 06" src="https://user-images.githubusercontent.com/3165635/208659417-25b5bb47-0c43-4022-a058-b272bcc90e5a.png">

Reply:
To see if the ids can break toolpad

<img width="220" alt="Screenshot 2022-12-20 at 11 44 57" src="https://user-images.githubusercontent.com/3165635/208659506-13c98a29-02cb-4cbd-ae0c-d10f65aafb20.png">